docs/skills.md

```
@@ -57,8 +59,33 @@ Use the skill inside a `/goal`:
+grok plugin install pr-shepherd --trust
+```
+
+Or register the server without the plugin:
+
+```bash
+grok mcp add pr-shepherd -- npx --yes --package pr-shepherd@<version> pr-shepherd-mcp
+```
+
+Use the skill from the slash menu:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)